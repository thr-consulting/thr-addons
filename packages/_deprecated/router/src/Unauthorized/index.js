/* eslint-disable global-require */
import React from 'react';
import {Grid, Header, Image} from 'semantic-ui-react';

export default function Unauthorized() {
	return (
		<Grid style={{marginTop: '2rem !important'}} container relaxed centered stackable columns="equal">
			<Grid.Column verticalAlign="middle" width="5">
				<Header as="h2">Not Authorized</Header>
				<p>You have insufficient permissions to access this page.</p>
			</Grid.Column>
			<Grid.Column textAlign="center" width="5">
				<Image size="small" src={require('./police.png')} alt="No Authorized"/>
			</Grid.Column>
		</Grid>
	);
}
