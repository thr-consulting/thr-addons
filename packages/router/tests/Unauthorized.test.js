import React from 'react';
import {shallow} from 'enzyme';
import Unauthorized from '../src/components/Unauthorized/index';

describe('Unauthorized', () => {
	it('should render without throwing an error', () => {
		expect(shallow(<Unauthorized/>)).toMatchSnapshot();
	});
});
