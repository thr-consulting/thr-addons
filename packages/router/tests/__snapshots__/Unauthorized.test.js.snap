// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unauthorized should render without throwing an error 1`] = `
<Styled(Grid)
  centered={true}
  columns="equal"
  container={true}
  relaxed={true}
  stackable={true}
>
  <GridColumn
    verticalAlign="middle"
    width="5"
  >
    <Header
      as="h2"
    >
      Not Authorized
    </Header>
    <p>
      You have insufficient permissions to access this page.
    </p>
  </GridColumn>
  <GridColumn
    textAlign="center"
    width="5"
  >
    <Image
      alt="No Authorized"
      as="img"
      size="small"
      src="test-file-stub"
      ui={true}
    />
  </GridColumn>
</Styled(Grid)>
`;
